tests/test_get_content.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from application.utils import get_content
+
+
+def test_adds_quotes():
+    directory = os.path.abspath(os.path.dirname(__file__))
+    csv_with_no_quotes = os.path.join(directory, "test_data/csv_with_no_quotes.csv")
```

Comment:
I'm also kind-of interested to see what happens if the "no quotes" csv has values that contain quotes... :see_no_evil:

Reply:
@TheDoubleK it works fine (the quotes don't get re-added). I've added an extra tests for this just to make sure though.